package.json

```diff
@@ -10,10 +10,12 @@
     "test": "./tst.sh",
     "t": "./tst.sh",
     "compile": "./cmp.sh",
-    "compileSol6": "./cmpSol6.sh",
```

Comment:
note:
Above version should be 2.0.0

not sure why I can't comment.
Seems need to reopen PR with new master.


also not sure why main is truffle.js
should probably change.